parser.y

```diff
@@ -7805,11 +7888,14 @@ TableRef:
 |	JoinTable
 
 TableFactor:
-	TableName PartitionNameListOpt TableAsNameOpt IndexHintListOpt
+	TableName PartitionNameListOpt TableAsNameOpt IndexHintListOpt TableSampleOpt
```

Comment:
should append a warning? since we don't support them all in TiDB actually...

Reply:
Related work is done in TiDB.